patroni/dcs/zookeeper.py

```diff
@@ -114,7 +114,7 @@ def set_ttl(self, ttl):
             return True
 
     def set_retry_timeout(self, retry_timeout):
-        self._client._retry.deadline = retry_timeout
+        self._client.retry.deadline = retry_timeout
```

Comment:
It seems to do the job.

kazoo 1.3.1 (debian 8):
```
>>> from kazoo.client import KazooClient, KazooRetry
>>> c = KazooClient()
>>> retry = c.retry if isinstance(c.retry, KazooRetry) else c._retry
>>> retry.deadline = 1
>>> retry.deadline
1
```
kazoo 2.2.1 (debian 9):
```
>>> from kazoo.client import KazooClient, KazooRetry
>>> c = KazooClient()
>>> retry = c.retry if isinstance(c.retry, KazooRetry) else c._retry
>>> retry.deadline = 1
>>> retry.deadline
1
```
Pushing a new commit. Thanks.